specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,3923 @@
+        },
+        "clusterId": {
+          "description": "The identity",
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true
+        },
+        "hosts": {
+          "description": "The hosts",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ClusterProperties": {
+      "description": "The properties of a cluster",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ManagementCluster"
+        }
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "The state of the cluster provisioning",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Deleting",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ClusterProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "storagePolicy": {
+          "description": "The storage policy of the cluster",
+          "$ref": "#/definitions/StoragePolicy"
+        }
+      }
+    },
+    "PrivateCloudList": {
+      "description": "A paged list of private clouds",
+      "properties": {
+        "value": {
+          "description": "The items on the page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/PrivateCloud"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClusterList": {
+      "description": "A paged list of clusters",
+      "properties": {
+        "value": {
+          "description": "The items on a page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AdminCredentials": {
+      "description": "Administrative credentials for accessing vCenter and NSX-T",
+      "properties": {
+        "nsxtUsername": {
+          "description": "NSX-T Manager username",
+          "type": "string",
+          "readOnly": true
+        },
+        "nsxtPassword": {
+          "description": "NSX-T Manager password",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-secret": true
+        },
+        "vcenterUsername": {
+          "description": "vCenter admin username",
+          "type": "string",
+          "readOnly": true
+        },
+        "vcenterPassword": {
+          "description": "vCenter admin password",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-secret": true
```

Comment:
`x-ms-secret` should probably be removed here to fix the check: `message: 'Secret property "vcenterPassword", cannot be sent in the response.'`.